Lib/glyphsLib/builder/hints.py

```diff
@@ -49,8 +50,14 @@ def to_glyphs_hints(self, ufo_glyph, layer):
         for attr in ["horizontal", "options", "stem", "type"]:
             setattr(hi, attr, hint[attr])
         for attr in ["origin", "other1", "other2", "place", "scale", "target"]:
-            # FIXME: (jany) what about target = up/down?
             if attr in hint:
-                value = Point(*hint[attr])
+                # handle target = ['u', 'p'] or ['d', 'o', 'w', 'n']
```

Comment:
@moyogo why do we need to `handle target = ['u', 'p'] or ['d', 'o', 'w', 'n']` ? Is that for backward compat with how these attribute was written out to UFO before this PR? If so, maybe add a comment with a link to the issue